cmd/api/auth/authn_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package auth
+
+import (
+	"context"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/authentication/v1"
```

Comment:
```suggestion
	apiAuthn "k8s.io/api/authentication/v1"
```

Calling this package just "v1" is going to be confusing. I suggest "apiAuthn" to make it clear what package this is.

Reply:
i think we should still keep `v1` in the name. using @beerparty's example: `apiAuthnv1`

Reply:
I've changed it to Sam's proposal in this and other cases. I've chosen to include the `api` part as there is a `client` part (`clientAuthnv1 "k8s.io/client-go/kubernetes/typed/authentication/v1"`). Done this change in other similar cases throughout the `api` module.